site/class-1-bar/youmghar/readme.md

```diff
@@ -0,0 +1,40 @@
+# Barchart assignment
+
+### This is a basic barchart with a hover which changes the colors of the bars.
+
+##Background
+
+This barchart works with D3. And loads external data into its HTML. Which is styled with a seperate CSS file. All the visuals are build up out off SVG.
+
+##Data
+
+The data that is visualized in the barchart is just test data. The data consists out of A letter which is placed on the x axis and a frequency which is shown on the y axis.
+
+##Features
+
+D3 API = https://d3js.org/d3.v4.min.js
+
+##License
+
+MIT License
```

Comment:
You’re missing citations to the original example you took the code from! I’m guessing you didn’t write this whole thing yourself?

And the readme assignment contained this text:

> 💁 The work you based your project on probably contains a license too. You cannot pick any license you’d want and disregard their license. Your license must be compatible with theirs. As a general rule of thumb: if you may create a derivative work (which is often true for free software licenses if your work is non-commercial) you can license your work under the same license as the original work.

Are you sure your able to license this code as MIT?